documentation/api/query/v4/aggregate-event-counts.markdown

```diff
@@ -1,5 +1,5 @@
 ---
-title: "PuppetDB 3.2 » API » v4 » Querying Aggregate Event Counts"
+title: "PuppetDB 3.2: Aggregate event counts endpoint"
```

Comment:
My thought here is that the user will generally navigate to these pages through the TOC, where the version number is clearly listed. When v5 is released, a second set of v5 docs can be added to the TOC, making it easy for users to select the appropriate version and/or see where they are in the docs set. 

All that being said, we can of course add the version number back to these page headings if you'd rather. In that case, however, should v1 be added to all the other API docs pages?